terraform/terraform.go

```diff
@@ -124,6 +125,9 @@ func getTerraformModulesDirPaths(dir string) []string {
 
 	for _, module := range modulesJson.Modules {
 		modulePath, err := filepath.EvalSymlinks(dir + "/" + module.Dir)
+		if os.IsNotExist(err) {
+			continue
```

Comment:
So this means that we wouldn't tag any of resource under the TF module since we can't find it.
I guess this is the most legitimate action indeed, but I'd add some warning log to indicate that

Reply:
@yaronya  something like this: (?)
cb72050ff931c47ac0220670ce57da039a9a28c5